test/functional/apps/dashboard/dashboard_clone.ts

```diff
@@ -40,7 +40,8 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.dashboard.loadSavedDashboard(clonedDashboardName);
       await retry.try(async () => {
         const panelTitles = await PageObjects.dashboard.getPanelTitles();
-        expect(panelTitles).to.eql(PageObjects.dashboard.getTestVisualizationNames());
+        const visiblePanelTitles = panelTitles.map((title) => title.split('\n')[1]);
```

Comment:
@constancecchen I wrote a helper method called `getVisiblePanelTitles()` after I wrote this string split a second or third time. I think we can clean these up to use that method (assuming tests still pass) but I wanted to get to green before refactoring.

Reply:
Just a hunch, but I think many these issues/functional failures will go away if we backport a fix to our EuiScreenReaderOnly change: https://github.com/elastic/eui/pull/5215